pages/calling_conventions.adoc

```diff
@@ -155,11 +155,11 @@ A function calling this needs to have at least 32 bytes of stack to store the
 parameters and align the stack upon call.
 
 ==== Return value
-If the return value is an integer/struct/union whose size is less than or equal
-than 64 bits, it is returned in `RAX`; otherwise, the struct is allocated by the
-caller and a pointer to it is passed as the first parameter, similarly to the
+If the return value is an integer, struct, or union whose size is less than or
+equal than 64 bits, it is returned in `RAX`; otherwise, the struct is allocated
+by the caller and a pointer to it is passed as the first parameter, like the
 Microsoft x64 ABI.
-Dissimilarly, the pointer is actually returned in `RAX` upon return.
+Unlike the Microsoft ABI, the pointer is actually returned in `RAX` upon return.
 
 [source,c]
 ---
```

Comment:
(unrelated to your changes, but curiosity) might be nice to check whether the ABI is referenced in this document, and adding a reference to it if not

Reply:
Both calling conventions seem to be referenced.